<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioSine</class>
 <widget class="QWidget" name="AudioSine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="audio_settings">
     <property name="text">
      <string>Audio settings</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Glow</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Thickness</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Repeat</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Oscillation</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="color_mode"/>
      </item>
      <item row="2" column="1">
       <widget class="QTooltipedSlider" name="thickness">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Color mode</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QFrame" name="colorsFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Background</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ColorPicker" name="background" native="true"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="wave_colorLabel">
           <property name="text">
            <string>Wave color</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="ColorPicker" name="wave_color" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTooltipedSlider" name="oscillation">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QTooltipedSlider" name="color_change_speed">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Cycle speed</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTooltipedSlider" name="repeat">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTooltipedSlider" name="glow">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTooltipedSlider</class>
   <extends>QSlider</extends>
   <header location="global">QTooltipedSlider.h</header>
  </customwidget>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QWidget</extends>
   <header>ColorPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>audio_settings</tabstop>
  <tabstop>repeat</tabstop>
  <tabstop>glow</tabstop>
  <tabstop>thickness</tabstop>
  <tabstop>oscillation</tabstop>
  <tabstop>color_mode</tabstop>
  <tabstop>color_change_speed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
